langchain/vectorstores/weaviate.py

```diff
@@ -42,6 +45,7 @@ def __init__(
         client: Any,
         index_name: str,
         text_key: str,
+        embedding: Embeddings,
```

Comment:
this isnt backwards compatible. lets make this optional, and then raise error if a method that requires this is used when its not provided

Reply:
done